classes/dojo.py

```diff
@@ -19,7 +19,6 @@ def __init__(self):
         self.living_spaces = []
         self.those_allocated_offices = []
         self.those_allocated_living_spaces = []
-        self.unallocated_persons = []
```

Comment:
I realised I wasn't using it anywhere in my code.
I would have used it for my print_unallocated method, but I took a different approach.